pycheribuild/utils.py

```diff
@@ -156,11 +158,13 @@ def _make_called_process_error(retcode, args, *, stdout=None, stderr=None, cwd=N
 
 def check_call_handle_noexec(cmdline: "typing.List[str]", **kwargs):
     try:
-        return subprocess.check_call(cmdline, **kwargs)
+        with keep_terminal_sane():
```

Comment:
I guess I could also do this at the start of __main__ to avoid some (minor overhead), but this way it should not mess up output if something goes wrong in the middle of a cheribuild execution.